string jsonDic
---
int8 success